test/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -125,6 +125,19 @@ public static DataFrame MakeDataFrameWithNumericAndStringColumns(int length, boo
             return df;
         }
 
+        public static DataFrame MakeDataFrameWithNumericStringAndDateTimeColumns(int length, bool withNulls = true)
+        {
+            DataFrame df = MakeDataFrameWithNumericAndStringColumns(length, withNulls);
+
+            DataFrameColumn dateTimeColumn = new PrimitiveDataFrameColumn<DateTime>("DateTime", Enumerable.Range(0, length).Select(x => DateTime.Today.AddDays(x)));
```

Comment:
It would probably be better if this returned constant data, no matter when the test is run.

So maybe hard-code a date (like June 3, 2021) and add days from there.

Reply:
Here's a concrete reason why:

If these tests run right close to midnight, they are going to fail. If `MakeDataFrameWithNumericStringAndDateTimeColumns` runs before midnight, `DateTime.Today` is going to return one day, but then in the test that calls this method, if it runs after midnight, `DateTime.Today` is going to return a different day, and now the test fails.

Reply:
Fixed